lib/candy-machine/constants.ts

```diff
@@ -49,6 +50,27 @@ export const CONFIG_ARRAY_START_V2 =
     32 +
     1 // gatekeeper
 
+export const CONFIG_ARRAY_START =
```

Comment:
What do you need this for?

Reply:
I used them before using the metaplex-foundation dependency and I forgot to delete them, now I correct it